routes/web.php

```diff
@@ -14,5 +16,25 @@
 */
 
 Route::get('/', function () {
-    return view('welcome');
-});
+    return view('/posts');});
+
+
+//Resource Controller
+Route::resource('posts', PostController::class);
```

Comment:
Please remove. No need for this since you're already using/overriding the actions below manually.